### ISSUE TITLE
Turn archive-get-queue-max into a size option

### ISSUE THREAD

**Issue by rdunklau** [pgbackrest/pgbackrest#623]
1. pgBackRest version:

pgBackRest 2.08dev (commit hash: b0659278cc7ab45519d791c10edbb494024f5148)

2. PostgreSQL version:
N/A

3. Operating system/version - if you have more than one server (for example, a database server, a repository host server, one or more standbys), please specify each:
N/A

4. Did you install pgBackRest from source or from a package?
Source

7. Describe the issue:
The documentation states that archive-get-queue-max can be entered as a size, just like archive-push-queue-max. However, the code currently defines it as an integer option in **build/lib/pgBackRestBuild/Config/Data.pm**

```
    &CFGOPT_ARCHIVE_PUSH_QUEUE_MAX =>
    {
        &CFGDEF_SECTION => CFGDEF_SECTION_GLOBAL,
        &CFGDEF_TYPE => CFGDEF_TYPE_SIZE,
        &CFGDEF_REQUIRED => false,
        &CFGDEF_NAME_ALT =>
        {
            'archive-queue-max' => {},
        },
        &CFGDEF_ALLOW_RANGE => [0, 4 * 1024 * 1024 * 1024 * 1024 * 1024], # 0-4PB
        &CFGDEF_COMMAND =>
        {
            &CFGCMD_ARCHIVE_PUSH => {},
        },
    },

    &CFGOPT_ARCHIVE_GET_QUEUE_MAX =>
    {
        &CFGDEF_SECTION => CFGDEF_SECTION_GLOBAL,
        &CFGDEF_TYPE => CFGDEF_TYPE_INTEGER,
        &CFGDEF_DEFAULT => 128 * 1024 * 1024, # 128MB
        &CFGDEF_ALLOW_RANGE => [0, 4 * 1024 * 1024 * 1024 * 1024 * 1024], # 0-4PB
        &CFGDEF_COMMAND =>
        {
            &CFGCMD_ARCHIVE_GET => {},
        },
    },
```

**Comment by dwsteele** [comment]
Whoops, that's definitely an oversight.  We'll fix that for the next release and check for any other options that should have size type.

**Comment by dwsteele** [comment]
Fixed at 64b97fd7ca3410cd6b.

I checked and didn't see any other options that should have had the size type.
